packages/smooth_app/lib/themes/smooth_theme.dart

```diff
@@ -124,7 +124,7 @@ class SmoothTheme {
     ),
     subtitle1: TextStyle(
       fontSize: 14.0,
-      fontWeight: FontWeight.w200,
+      fontWeight: FontWeight.w500,
```

Comment:
Hi @M123-dev!
I think `FontWeight.w500` is the same as the default `FontWeight.normal` value. Which is the same as nothing, as it is the default.
IMO that would make sense to remove this line altogether.

Reply:
Just checked w400 is the default, but its probably best to use the "normal" value